tests/integration/files/file/base/issue-30161.sls

```diff
@@ -1,24 +1,44 @@
 unless_false_onlyif_true:
   file.managed:
     - name: {{ salt['runtests_helpers.get_salt_temp_dir_for_path']('test.txt') }}
+{% if grains['os'] != 'Windows' %}
```

Comment:
This logic feels weird. Since we're launching Windows specific-commands, shouldn't we be checking _for_ Windows. That might future-proof this a bit more.